storage/pom.xml

```diff
@@ -44,6 +44,23 @@
       <artifactId>grpc-netty</artifactId>
       <version>1.0.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>commons-logging</groupId>
+      <artifactId>commons-logging</artifactId>
+      <version>1.1.1</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>pl.pragmatists</groupId>
+      <artifactId>JUnitParams</artifactId>
```

Comment:
personally prefer assertj, and it is used in all zipkin java projects, but by no means required. just sayin